### PR TITLE
BF Notification Update

### LINKED ISSUE #2189: [SUGGESTION] Battle Frontier Notification: Include Money
**Current Behavior**

The notification I get from Battle Frontier after I lose only states the stage I completed and the amount of BP I gained.

**Improved Behavior**

Show what it does now, with the addition of how much money I got as well.

(As a side note, would be nice if the notification stays up longer than some minutes. Battle Frontier takes so long that I tend to do something else while it runs, and I almost never see the notification because it doesn't stay around long enough.)

**Reason for change**

I know it's a small thing, but it beats me having to calculate the amount each time. Especially when I'm spending money while I'm playing Battle Frontier resetting quests to ones that are doable while playing Battle Frontier.

**Detailed Description**

**Screenshots:**

### PR DESCRIPTION
**Pull request by jk13pclick**
Closes #2189. Untested, but should work fine

### CODE REVIEW

File: src/scripts/battleFrontier/BattleFrontierRunner.ts [hunk 1]
```diff
@@ -67,10 +67,10 @@ class BattleFrontierRunner {
 
         Notifier.notify({
             title: 'Battle Frontier',
-            message: `You managed to beat stage ${stageBeaten}.\nYou received ${battlePointsEarned} BP`,
+            message: `You managed to beat stage ${stageBeaten}.\nYou received ${battlePointsEarned} BP.\nYou received ${moneyEarned}.`,
```
Comment (RedSparr0w) [inline src/scripts/battleFrontier/BattleFrontierRunner.ts:70]: There should also be something to identify that the 2nd amount is money,
Any one of these should do:
```
<img src='moneyimagelocation'/> xxx
xxx money (or pokedollars, don't remember what we call it 😅)
$xxx
```
Could you also `toLocaleString` the amounts (should probably also be done to the BP amount).

**Comment by jk13pclick** [comment]
Should be good now, tested and works!

File: src/scripts/battleFrontier/BattleFrontierRunner.ts [hunk 2]
```diff
@@ -67,10 +67,10 @@ class BattleFrontierRunner {
 
         Notifier.notify({
             title: 'Battle Frontier',
-            message: `You managed to beat stage ${stageBeaten}.\nYou received ${battlePointsEarned} BP`,
+            message: `You managed to beat stage ${stageBeaten}.\nYou received ${battlePointsEarned.toLocaleString('en-US')} BP <img src="./assets/images/currency/battlePoint.svg" height="18px"/>.\nYou received ${moneyEarned.toLocaleString('en-US')} PokéDollars <img src="./assets/images/currency/money.svg" height="18px"/>.`,
```
[the remark this answers is not in the document]
Comment (jk13pclick) [reply src/scripts/battleFrontier/BattleFrontierRunner.ts:70]: We just use the image for Offline Earnings, I'll go with that

**Review by RedSparr0w — APPROVED** [review]
Awesome, looks good, thanks for the PR! 👍

**Pull request merged**